tests/func/test_api.py

```diff
@@ -83,6 +85,10 @@ def test_open(tmp_dir, dvc, remote):
     ],
 )
 def test_open_external(erepo_dir, cloud):
+    if isinstance(cloud, SSHMocked) and platform.system() == "Windows":
```

Comment:
You could just use `pytest.param(pytest.lazy_fixture("ssh"), marks=pytest.mark.xfail(...))` in `@parametrize` above.

Reply:
ping

Reply:
sorry, I did mark in  parametrization and did not remove this check